index.js

```diff
@@ -98,10 +101,13 @@ const copyCacheToModule = async (moduleExecutablePath, platform, revision) => {
 }
 
 module.exports = async ({
-  platform: platform = currentPlatform,
-  revision: revision = '499413',
-  log: log = false
-} = {}) => {
+                          platform: platform = currentPlatform,
+                          revision: revision = '499413',
+                          log: log = false,
+                          installPath: installPathBase = __dirname,
+                          onProgress: onProgress
+                        } = {}) => {
```

Comment:
indentation is off, please stick to 2 spaces

Reply:
done